VideoWeb/VideoWeb.AcceptanceTests/Steps/HearingRoomSteps.cs

```diff
@@ -167,8 +167,11 @@ public void ProgressToNextPage()
         private void SwitchToTheJudgeIFrame()
         {
             if (_c.Test.JudgeInIframe) return;
-            if (!_browsers[_c.CurrentUser.Key].IsDisplayed(By.Id(HearingRoomPage.JudgeIframeId))) return;
-            _browsers[_c.CurrentUser.Key].Driver.SwitchTo().Frame(HearingRoomPage.JudgeIframeId);
+            if (!_browsers[_c.CurrentUser.Key].IsDisplayed(HearingRoomPage.PauseButton))
+            {
+                _browsers[_c.CurrentUser.Key].Driver.WaitUntilVisible(ClerkWaitingRoomPage.StartVideoHearingButton).Displayed.Should().BeTrue();
+                _browsers[_c.CurrentUser.Key].Driver.FindElement(ClerkWaitingRoomPage.StartVideoHearingButton).Click();
```

Comment:
don't use the native web driver .Click() as it will fail on Safari. Instead us _browsers[_c.CurrentUser.Key].Click(element);

Reply:
done